yarn.lock

```diff
@@ -14511,6 +14511,15 @@ string-length@^4.0.1:
     is-fullwidth-code-point "^2.0.0"
     strip-ansi "^4.0.0"
 
+"string-width@^1.0.2 || 2 || 3 || 4", string-width@^4.0.0, string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.2, string-width@^4.2.3:
```

Comment:
What happened with all this string-width and strip-ansi stuff?

Reply:
I don't know...
It just divided the same dependency into multiple parts, I don't where it came from.

I reverted it.